src/OrbitGl/CallStackDataView.cpp

```diff
@@ -161,7 +161,7 @@ void CallStackDataView::DoFilter() {
     return;
   }
 
-  std::vector<uint32_t> indices;
+  std::vector<uint64_t> indices;
```

Comment:
One view - LiveFunctionView is using ids with map instead of a vector with index (which reminds me that for the vector with an index this should also be uint64_t on lp64). This change is related to change in LiveFunctionsDataView